default-vlc-path.patch

```diff
@@ -0,0 +1,13 @@
+diff --git i/src/main/java/mediathek/tool/GuiFunktionenProgramme.java w/src/main/java/mediathek/tool/GuiFunktionenProgramme.java
+index 5f25443..6a1d737 100644
+--- i/src/main/java/mediathek/tool/GuiFunktionenProgramme.java
++++ w/src/main/java/mediathek/tool/GuiFunktionenProgramme.java
+@@ -77,7 +77,7 @@ public static String getPathToApplicationJar() {
+         return s;
+     }
+ 
+-    private static final String PFAD_LINUX_VLC = "/usr/bin/vlc";
++    private static final String PFAD_LINUX_VLC = "/app/bin/vlc";
```

Comment:
This somehow feels wrong, as you will need to instruct user they need to install VLC from Flathub. There's no way to express cross-apps dependencies with flatpak.

Reply:
I think I asked this before, but… what else should I do?